pkg/requests/webhook_endpoints.go

```diff
@@ -40,3 +42,39 @@ func WebhookEndpointsList(ctx context.Context, baseURL, apiVersion, apiKey strin
 
 	return data
 }
+
+// WebhookEndpointCreate creates a new webhook endpoint
+func WebhookEndpointCreate(ctx context.Context, baseURL, apiVersion, apiKey, url, description string, connect bool, profile *config.Profile) error {
+	if strings.TrimSpace(url) == "" {
+		return fmt.Errorf("url cannot be empty")
+	}
+
+	data := []string{
+		fmt.Sprintf("url=%s", url),
+		"enabled_events[]=*",
+		"api_version=''", // empty string so the new webhook endpoint uses the merchant default version
```

Comment:
I just tried this out with curl, and It seems like `api_version=""` may not work. OTOH, omitting `api_version` seems to work. Maybe we could have the caller pass an empty string, and this function check `if apiVersion != ""` like we're doing for `description`?

Reply:
i think we can leave it out entirely, because it doesnt make sense for users to create a webhook endpoint for an api version thats different from their merchant account (especially thru vscode ext)

Reply:
sgtm!

Reply:
i justed tested creating a new webhook endpoint without specifyin the api version. the new endpoint is created with my merchant api